From:dsanders (Daniel Sanders)
https://reviews.llvm.org/D13027
[lldb-mi] Fix unresolved reference to llvm_regcomp and llvm_regfree.
----
diff -urp llvm-3.9.0.src.orig/tools/lldb/tools/lldb-mi/CMakeLists.txt llvm-3.9.0.src/tools/lldb/tools/lldb-mi/CMakeLists.txt
--- llvm-3.9.0.src.orig/tools/lldb/tools/lldb-mi/CMakeLists.txt	2016-03-29 07:39:10.000000000 -0700
+++ llvm-3.9.0.src/tools/lldb/tools/lldb-mi/CMakeLists.txt	2017-01-24 16:46:00.073055160 -0800
@@ -82,6 +82,8 @@ if ( CMAKE_SYSTEM_NAME MATCHES "Windows"
     )
 endif ()
 
+include(../../cmake/LLDBDependencies.cmake)
+
 add_lldb_executable(lldb-mi ${LLDB_MI_SOURCES})
 
 target_link_libraries(lldb-mi liblldb)
Only in llvm-3.9.0.src/tools/lldb/tools/lldb-mi: CMakeLists.txt.orig
Only in llvm-3.9.0.src/tools/lldb/tools/lldb-mi: CMakeLists.txt.rej
